style.css

```diff
@@ -505,6 +513,60 @@ hr {
   flex-wrap: wrap;
 }
 
-.footer p {
+.footer {
+  background-color: #464646;
+}
+
+.footer p,
+a {
   margin: 0 0.25em;
+  color: #ffffff;
+  font-family: Al Nile;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+}
+.footer .credits {
+  margin: auto;
+  order: 2;
```

Comment:
Why give things an explicit order?

Reply:
That's the solution I found for the problem.

Reply:
https://stackoverflow.com/questions/40910851/change-div-position-depending-on-screen-resolution-flexbox